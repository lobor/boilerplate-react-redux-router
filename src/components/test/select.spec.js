import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Select from './../select'

test('render Select', () => {
  const tree = renderer.create(<Select />).toJSON()
  expect(tree).toMatchSnapshot()
})
