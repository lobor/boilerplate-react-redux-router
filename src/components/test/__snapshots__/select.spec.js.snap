// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Select 1`] = `
.c0 {
  height: 38px;
  padding: 6px 10px;
  background-color: #fff;
  border: 1px solid #D1D1D1;
  border-radius: 4px;
  box-shadow: none;
  box-sizing: border-box;
}

.c0:focus {
  border: 1px solid #33C3F0;
  outline: 0;
}

<select
  className="c0"
/>
`;
